src/components/ButtonPanel/ButtonPanel.js

```diff
@@ -6,38 +6,43 @@ const buttonPanel = ({ clickHandler }) => {
 
   return (
     <div>
-      <div className = "group group-1">
+
+    // <div className = "button" style = {style} onClick ={handleClick}>
+    //   <h3>{props.name}</h3>
+    // </div>
```

Comment:
Please do not leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.